### PR TITLE
followup displays email which will be sent to authors

### PR DESCRIPTION
**Pull request by Trim**
While updating the CSS, I noticed the padding around the content
were not removed for small screens in the redaction space.

It's fixed too.

See [suivi request](https://linuxfr.org/suivi/message-pour-relance-dans-l-espace-de-redaction#comment-1853867).

Example of rendering for followup form:

relancer les rédacteurs

### CODE REVIEW

File: app/views/redaction/news/show.html.haml [hunk 1]
```diff
@@ -41,13 +41,21 @@
   = button_to "Effacer la dépêche", erase_redaction_news_path(@news), class: "erase_news", data: { confirm: "Confirmez‑vous vouloir supprimer cette dépêche ?" }
 - if current_account.can_followup? @news
   %button#followup.more Relancer les rédacteurs
-  .more_actions(style="display: none;")
+  #followup_container.more_actions(style="display: none;")
+    %h1 Relancer les rédacteurs
+    %p
+      Vous pouvez envoyer un email de relance aux participants de cette dépêche
```
Comment (Oumph) [inline app/views/redaction/news/show.html.haml:47]:
```suggestion
      Vous pouvez envoyer un courriel de relance aux participants de cette dépêche
```

File: app/views/redaction/news/show.html.haml [hunk 2]
```diff
@@ -41,13 +41,21 @@
...
+      pour leur demander de la finaliser.
+    %p
+      N’hésitez pas à leur donner des précisions (date de sortie imminente, demande de précisions, etc.).
```
Comment (Oumph) [inline app/views/redaction/news/show.html.haml:50]:
```suggestion
      N’hésitez pas à leur donner ou demander des précisions (date de sortie imminente, demande d’informations complémentaires, etc.).
```

File: app/views/redaction/news/show.html.haml [hunk 3]
```diff
@@ -41,13 +41,21 @@
...
+    %h2 Email de relance
```
Comment (Oumph) [inline app/views/redaction/news/show.html.haml:51]:
```suggestion
    %h2 Courriel de relance
```

File: app/views/redaction/news/show.html.haml [hunk 4]
```diff
@@ -41,13 +41,21 @@
...
     = form_tag followup_redaction_news_path(@news) do
-      %p
-        Vous pouvez envoyer un message de relance aux participants de cette dépêche
-        pour leur demander de la finaliser. N’hésitez pas à leur donner des 
-        précisions (date de sortie imminente, demande de précisions, etc.).
-      = text_area_tag :message, nil, rows: 10
+      %article#followup_mail
+        %pre#follwoup_header
+          = render formats: [:text], partial: 'news_notifications/followup_header'
+        = text_area_tag :message, "L'équipe d'animation vous encourage à finaliser la dépêche.", rows: 10
```
Comment (Oumph) [inline app/views/redaction/news/show.html.haml:56]:
```suggestion
        = text_area_tag :message, "L’équipe d’animation vous encourage à finaliser la dépêche.", rows: 10
```

**Review by Oumph — CHANGES_REQUESTED** [review]

**Review by Trim — COMMENTED** [review]
Thanks for the review, I've applied commits to apply requested changes.

**Pull request merged**